ovs-dpdk.md

```diff
@@ -9,137 +9,102 @@ According to [Intel ONP performance tests](https://download.01.org/packet-proces
 OVS-DPDK has been added to oVirt as an experimental feature.
 Please note that OVS-DPDK cannot access any features located within the linux kernel, for example: linux bridge,  tun/tap devices, iptables etc.
 
-## Requirements for DPDK:
+## Requirements:
 
 For best performance use an Intel Xeon class server system such as Ivy Bridge, Haswell or newer.
 Ensure that each memory channel has at least one memory DIMM inserted, and that memory size should be at least 4GB.
  
 The network card must be on the supported-vendor matrix, located here: <http://dpdk.org/doc/nics>
 
+* Hardware support: Make sure VT-d / AMD-Vi is enabled in BIOS
+* Kerenel support: When adding a new host via oVirt GUI, go to Hosts -> Edit > Kernel and add the following parameters to Kernel Command line:  ```“iommu=pt intel_iommu=on default_hugepagesz=1GB hugepagesz=1G hugepages=N” ```
+Then hit the host maintenance mode, reboot the host and activate it when it is up again.
```

Comment:
hit the host maintenance mode -> "move the host to maintenance"
or "click host maintenance button" ?